client-frameworks-support/testing-utilities/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
```

Comment:
Maybe instead of the other `/dist/package.json`, Change here /dist/index.js to /index.js and to put the dist folder in gitignore altogether 🤔